Examples/mParticle-Branch-Example/mParticle/APEventViewController.swift

```diff
@@ -262,19 +262,21 @@ class APEventViewController: UIViewController, UITableViewDelegate, UITableViewD
 
     func finishSetIdentityAlias(alias: String) {
         let request = MPIdentityApiRequest.withEmptyUser()
-        request.setUserIdentity(alias, identityType: MPUserIdentity.other)
-        request.onUserAlias = { (previousUser, newUser) -> Void in
+        request.setIdentity(alias, identityType: MPIdentity.other)
 
-            //copy anything that you want from the previous to the new user
-            //this snippet would copy everything
-
-            newUser.userAttributes = previousUser.userAttributes
-
-            let products = previousUser.cart.products() ?? []
-            if (products.count > 0) {
-                newUser.cart.addAllProducts(products, shouldLogEvents: false)
-            }
-        }
+        // TODO: This has been deprecated and removed.   Switch to mParticleIdentityStateChangeListener
+//        request.onUserAlias = { (previousUser, newUser) -> Void in
+//
+//            //copy anything that you want from the previous to the new user
+//            //this snippet would copy everything
+//
+//            newUser.userAttributes = previousUser.userAttributes
+//
+//            let products = previousUser.cart.products() ?? []
+//            if (products.count > 0) {
+//                newUser.cart.addAllProducts(products, shouldLogEvents: false)
+//            }
+//        }
```

Comment:
The cart is no longer supported by the mParticle SDK. We recommend clients keep track of their own shopping carts and create commerce events as appropriate. So you could just go ahead and delete this code.

Reply:
@BrandonStalnaker our iOS SDK engineer is on vacation for a week. Can we push this change out now for iOS 14, then come and clean this up in the next iteration?